leptos/src/lib.rs

```diff
@@ -169,7 +169,7 @@ pub mod prelude {
         pub use oco_ref::*;
         pub use reactive_graph::{
             actions::*, computed::*, effect::*, owner::*, signal::*,
-            wrappers::read::*, *,
+            wrappers::read::*,
```

Comment:
Here `signal` is ambiguous. Running `cargo doc --no-deps -p leptos` shows:

```
warning: `leptos::prelude::signal` is both a function and a module
  --> leptos/src/lib.rs:74:29
   |
74 | //!   - signals: [`signal`](leptos::prelude::signal), [`ReadSignal`](leptos::prelude::ReadSignal),
   |                             ^^^^^^^^^^^^^^^^^^^^^^^ ambiguous link
   |
   = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default
help: to link to the function, add parentheses
   |
74 | //!   - signals: [`signal`](leptos::prelude::signal()), [`ReadSignal`](leptos::prelude::ReadSignal),
   |                                                    ++
help: to link to the module, prefix with `mod@`
   |
74 | //!   - signals: [`signal`](mod@leptos::prelude::signal), [`ReadSignal`](leptos::prelude::ReadSignal),
   |                             ++++
```

Seems to me that the `*` export from `reactive_graph` is really consusing.

Reply:
Might be worth not exporting the `signal()` (and depreciate it) and only export `Signal::new()`.

CC @gbj

Reply:
@rakshith-ravi `signal()` here is the only one that still primarily uses a function call, because it returns a tuple `(ReadSignal<T>, WriteSignal<T>)`, so it is named `signal()` like a channel is `channel()`, not `Signal::new()`